src/SamplesApp/UITests.Shared/Windows_Devices/Midi/MidiDeviceEnumeration.xaml.cs

```diff
@@ -0,0 +1,444 @@
+			if (devInfo == null)
+			{
+				InputDeviceProperties.Clear();
+				InputDeviceProperties.Add("Device not found!");
+				inputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			// Display the found properties
+			DisplayDeviceProperties(devInfo, inputDeviceProperties, InputDeviceProperties);
+			NotifyUser("Device information found!");
+		}
+
+		/// <summary>
+		/// Change the active output MIDI device
+		/// </summary>
+		/// <param name="sender">Element that fired the event</param>
+		/// <param name="e">Event arguments</param>
+		private void outputDevices_SelectionChanged(object sender, SelectionChangedEventArgs e)
+		{
+			// Get the selected output MIDI device
+			int selectedOutputDeviceIndex = outputDevices.SelectedIndex;
+
+			// Try to display the appropriate device properties
+			if (selectedOutputDeviceIndex < 0)
+			{
+				// Clear output device properties
+				OutputDeviceProperties.Clear();
+				OutputDeviceProperties.Add("Select a MIDI output device to view its properties");
+				outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Select a MIDI output device to view its properties");
+				return;
+			}
+
+			DeviceInformationCollection devInfoCollection = _midiOutDeviceWatcher.GetDeviceInformationCollection();
+			if (devInfoCollection == null)
+			{
+				OutputDeviceProperties.Clear();
+				OutputDeviceProperties.Add("Device not found!");
+				outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			DeviceInformation devInfo = devInfoCollection[selectedOutputDeviceIndex];
+			if (devInfo == null)
+			{
+				OutputDeviceProperties.Clear();
+				OutputDeviceProperties.Add("Device not found!");
+				outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			// Display the found properties
+			DisplayDeviceProperties(devInfo, outputDeviceProperties, OutputDeviceProperties);
+			NotifyUser("Device information found!");
+		}
+
+		/// <summary>
+		/// Display the properties of the MIDI device to the user
+		/// </summary>
+		/// <param name="devInfo"></param>
+		/// <param name="propertiesList"></param>
+		private void DisplayDeviceProperties(DeviceInformation devInfo, ListView propertiesList, ObservableCollection<string> items)
+		{
+			items.Clear();
+			items.Add("Id: " + devInfo.Id);
+			items.Add("Name: " + devInfo.Name);
+			items.Add("IsDefault: " + devInfo.IsDefault);
+			items.Add("IsEnabled: " + devInfo.IsEnabled);
+			items.Add("EnclosureLocation: " + devInfo.EnclosureLocation);
+
+			// Add device interface information
+			items.Add("----Device Interface----");
+			foreach (var deviceProperty in devInfo.Properties)
+			{
+				items.Add(deviceProperty.Key + ": " + deviceProperty.Value);
+			}
+
+			propertiesList.IsEnabled = true;
+		}
+
+		private void NotifyUser(string message)
+		{
+			statusBlock.Text = message;
+		}
+
+		//private byte ENote = 64;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)